go.mod

```diff
@@ -69,4 +69,4 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-replace golang.org/x/net => golang.org/x/net v0.4.0
+replace golang.org/x/net => golang.org/x/net v0.8.0
```

Comment:
Also in general, having replace statements is not going to work well with dependabot. Are we able to explicit set the version in the "require" section?

Reply:
This is an indirect dependency can we add it in require section? The other option I tried was to bump client-go to 1.24.17 as `golang/x/net` is it's dependency